addons/mail/static/src/core/common/thread.xml

```diff
@@ -102,14 +98,24 @@
 </t>
 
 <t t-name="mail.Thread.jumpPresent">
-    <span t-if="props.showJumpPresent and state.showJumpPresent" class="o-mail-Thread-jumpPresent position-sticky btn btn-link alert alert-info d-flex cursor-pointer align-items-center py-2 m-0" t-att-class="{ 'px-4': !env.inChatWindow, 'px-2': env.inChatWindow, 'top-0': props.order !== 'asc', 'bottom-0': props.order === 'asc' }" role="button" t-on-click="() => this.jumpToPresent()">
-        <span class="small">You're viewing older messages</span>
-        <span class="flex-grow-1"/>
-        <span class="fw-bolder small pe-2">Jump to Present</span>
-        <i class="fa" t-att-class="{ 'fa-caret-up': props.order !== 'asc', 'fa-caret-down': props.order === 'asc' }"/>
+    <span t-if="props.showJumpPresent and state.showJumpPresent" t-att-class="{
+        'mx-2 rounded-top': !env.inChatWindow and !env.inChatter,
+        'm-0 px-4 top-0': env.inChatter,
+        'bottom-0': !env.inChatter
+    }" class="o-mail-Thread-banner o-mail-hoverable position-sticky d-flex justify-content-between alert alert-primary border-0 rounded-0 mb-0 py-1 cursor-pointer shadow-sm small fw-bold" t-on-click="() => this.jumpToPresent()">
+        <span>You're viewing older messages</span>
+        <span>Jump to Present<i class="ms-2 fa" t-att-class="{ 'fa-caret-up': props.order !== 'asc', 'fa-caret-down': props.order === 'asc' }"/></span>
     </span>
 </t>
 
+<t t-name="mail.Thread.jumpUnread">
+    <span t-if="props.thread.selfMember?.localMessageUnreadCounter > 0" class="o-mail-Thread-banner position-sticky top-0 d-flex cursor-pointer shadow-sm small fw-bold" t-att-class="{'mx-2': !env.inChatWindow}">
+        <t t-set="alertClass" t-value="'alert alert-info m-0 o-mail-hoverable rounded-top-0 py-1'"/>
```

Comment:
I'm still not satisfied with the width of alerts:
<img width="983" alt="Screenshot 2024-05-29 at 17 18 12" src="https://github.com/odoo/odoo/assets/6569390/b04630b3-ca0e-44d8-a065-23db122588b7">

For the jump, other apps that have this visual ensure the size matches the composer. This is hard to do as it is, because this is not an extension of the composer, and it would look off to be offset due to avatar (and it looks bad above avatar, as it doesn't look like an extension of the composer visually

I really don't see the point not to take the whole width right now. The left/right spacing looks much noiser, because it's as if this add left and right invisible column, making the layout look more complex.

_(and visually they look like jaws, which is creepy 🐊)_

Reply:
![image](https://github.com/odoo/odoo/assets/48757558/17ca5a7b-5e0c-432a-8b89-37aa72170bd8)


Maybe a bit better, still a little odd in respect to the avatar in the composer.